tests/data_sources/test_nwp_data_source.py

```diff
@@ -64,7 +64,7 @@ def test_nwp_get_contiguous_time_periods():  # noqa: D103
 
     contiguous_time_periods = nwp.get_contiguous_time_periods()
     correct_time_periods = pd.DataFrame(
-        [{"start_dt": pd.Timestamp("2019-01-01 00:00"), "end_dt": pd.Timestamp("2019-01-02 02:00")}]
+        [{"start_dt": pd.Timestamp("2019-01-01 00:00"), "end_dt": pd.Timestamp("2019-01-02 04:00")}]
```

Comment:
The `end_dt` need to be changed because I removed `NWPDataSource.datetimes()` is now using all forecast timesteps, not just the first three forecast timesteps.